app/views/rides/_error_explanation.html.erb

```diff
@@ -1,11 +1,5 @@
 <% if !errors.nil? && errors.any? %>
-  <div id="error_explanation">
-    <h2><%= problem %> (<%= pluralize(errors.count, "error") %>):</h2>
-
-    <ul>
-      <% errors.full_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-    </ul>
-  </div>
+  <% errors.full_messages.each do |message| %>
+    <%= javascript_tag "M.toast({html: '#{message}'}, 4000)" %>
```

Comment:
Again, fishy looking.  I don't want HTML strings being rendered if we can avoid it

Reply:
Aren't Html strings being rendered in the current implementation? Is it a question of JavaScript vs not?

Reply:
The strings get escaped by the rails framework if they are rendered directly into a template.  The `javascript_tag` function prevents escaping, and the `#{...}` string interpolation doesn't do anything for us in terms of escaping.